modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -86,15 +86,16 @@ If you do not add both channels, you will not be able to create the bootstrap re
 +
 * For {es} 6:
 +
-From the {webui}, add [systemitem]``RHEL6 Base x86_64`` and [systemitem]``SUSE Linux Enterprise Client Tools RES6 x86_64``.
+From the {webui}, add [systemitem]``RHEL Expanded Support 6 x86_64``.
 +
-From the command prompt, add [systemitem]``rhel-x86_64-server-6`` and [systemitem]``res6-suse-manager-tools-x86_64``.
```

Comment:
The other two changes are exactly what I reported, but that line I'm not sure, I will need to check.
Give me some time, I will come back with it.

Reply:
As discussed during today's standup, @calancha will take care of that check.